components/op/src/config.rs

```diff
@@ -16,18 +16,18 @@
 pub struct Config {
     // file to hash
     pub file: Option<String>,
-    // origin to get the shard of
-    pub origin: Option<String>,
-    // account to get the shard of
-    pub account: Option<u64>,
+    // string to get the shard of
+    pub string: Option<String>,
+    // numeric id to get the shard of
+    pub id: Option<u64>,
 }
 
 impl Default for Config {
     fn default() -> Self {
         Config {
             file: None,
-            origin: None,
-            account: None,
+            string: None,
```

Comment:
string as a name just seems too generic.. can we just call it shard (String), and have it do the right thing if the string is actually a number (I don't _think_ we can have origins or names that are numbers only).

Reply:
Sure